pkg/cmdutil/userio/file_picker.go

```diff
@@ -53,10 +54,18 @@ func (ifp *FilePicker) GetInput(streams IOStreams) (string, error) {
 		expandedValue:  expandedValue,
 		styles:         streams.styles,
 	}
-	result, err := streams.execute(ifpModel)
-	if err != nil {
-		return "", err
+
+	// Allow nesting inside other components
+	var result tea.Model
+	if streams.CurrentHandler != nil {
+		result = streams.CurrentHandler.SetInputModel(ifpModel)
```

Comment:
feels odd that we have to repeat the same thing in every component. Ideally, components shouldn't care if they are executed withing wizard or not. First solution, which comes to my mind, make component accept an interface with a few required methods, like `execute`, `styles`, `colorScheme`, maybe something else

Reply:
I've solved this outside of the individual components now, no component changes required

Reply:
Should it be reverted in this case? I see similar changes in other components

Reply:
yep, done!